packages/core/admin/admin/src/utils/tests/hashAdminUserEmail.test.ts

```diff
@@ -10,6 +10,7 @@ describe('Creating admin user email hash in admin', () => {
     Object.defineProperty(global.self, 'crypto', {
```

Comment:
@joshuaellis These need to be added to the ambient types. Do you know how which type I could use so that we don't have to type it manually?